docs/CSS_POLICY.md

```diff
@@ -0,0 +1,53 @@
+# CSS Policy
+
+We use the [GOVUK.UK Design System][govuk-ds] to provide as much as our styling as possible:
+
+- To benefit from their accessibility, performance and usability concerns
+- To be in-line with other GOV.UK services
+- To reduce our maintenance requirements
+- To be compliant with GDS standards
+
+Where possible, **GOV.UK Design System components and patterns should be used**.
+
+## Custom app CSS
+
+Where the GOV.UK Design System does not meet our needs, we _may_ write custom CSS for the DigiDeps app. When doing so, the following rules should be followed:
+
+- Reusable components should be designed as Twig templates or macros, so they can easily be updated in the future
+- CSS should use classes, rather than IDs, attributes or tag selectors
+- All custom CSS classes should be prefixed `app-`, to differentiate them from GOV.UK classes which are prefixed `govuk-`
```

Comment:
app- is a bit generic and may end up colliding elsewhere. Might be worth going more specific, i.e. opg- or opg-dd- allows for reuse from other opg sites on gov.uk.

Reply:
Good call, that was from my unnuanced read of the [style guide](https://github.com/alphagov/govuk-frontend/blob/1ea024f26834b5059774c207d1ad8340a6c0ce25/docs/contributing/coding-standards/css.md#user-content-govuk-namespacing).

I think `opg-` makes most sense for portability to other applications. It's also in-line with what Serve OPG do, so helps for future alignment.